packages/feedback/components/feedback/src/feedback.scss

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright IBM Corp. 2023, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+$css--plex: true !default;
+
+@use '../../../../../globals/scss/vars' as *;
+@use '@carbon/styles/scss/theme' as *;
+
+:host(#{$c4ai-prefix}-feedback) hr {
+  border: 0;
+  background: var(--cds-border-subtle);
```

Comment:
You should be able to access the carbon tokens directly

`background: $border-subtle;`

Reply:
Done